pkg/server/backend_manager.go

```diff
@@ -259,10 +278,10 @@ func (s *DefaultBackendStorage) RemoveBackend(identifier string, idType header.I
 	}
 	var found bool
 	for i, c := range backends {
```

Comment:
nit (since backends are no longer just connection objects)
```suggestion
	for i, b := range backends {
```

Reply:
Done